src/wp-includes/option.php

```diff
@@ -918,20 +927,22 @@ function update_option( $option, $value, $autoload = null ) {
  *
```

Comment:
In response to https://github.com/WordPress/wordpress-develop/pull/5671/files#r1534716638:
```suggestion
 * @since 6.6.0 The $autoload parameter's default value was changed to null.
 *
```